### PR TITLE
There were no bugs in ba sing se

### PR DESCRIPTION
**Pull request by Aegyo**
Untested but I'm pretty sure it will work, literally copied and pasted nothing can go wrong right??

### CODE REVIEW

**Comment by Aegyo** [comment]
ok I tested and it doesn't work, one moment pls

File: other/quiz/quiz.js [hunk 1]
```diff
@@ -115,6 +115,11 @@ const newQuiz = async (guild, reoccur = false) => {
 
   const incorrectFilter = quiz_channel.createMessageCollector({ filter: (m) => !filter(m), time: time_limit});
   incorrectFilter.on('collect', async m => {
+    // stop doing emoji when we're already done pls thx you
+    if (winners.has(user.id) || m.createdTimestamp - finished > ANSWER_TIME_LIMIT) {
```
Comment (RedSparr0w) [inline other/quiz/quiz.js]: I think this just won't run, as finished is `0` beforehand, need to wait until finished is defined?
```suggestion
    if (finished && (winners.has(user.id) || m.createdTimestamp - finished > ANSWER_TIME_LIMIT)) {
```

**Review by RedSparr0w — APPROVED** [review]
Looks good 🎉

File: other/quiz/quiz.js (hunk 1 above)
Comment (Aegyo) [reply other/quiz/quiz.js]: did I update while you were reviewing?

Comment (RedSparr0w) [reply other/quiz/quiz.js]: Yeah must have, koz when I reloaded the changes where already fixed lol

**Pull request merged**